kiwi/storage/subformat/vagrant_virtualbox.py

```diff
@@ -0,0 +1,432 @@
+#
+
+import os
+
+# project
+from kiwi.storage.subformat.vagrant_base import DiskFormatVagrantBase
+from kiwi.storage.subformat.vmdk import DiskFormatVmdk
+from kiwi.command import Command
+
+
+# Template from which the vagrant boxes' ovf file is created
+#
+# This file has been extracted from a vagrant box that was built with
+# packer. The variable parts are marked via python format strings and were taken
+# from a script provided by Neal Gompa.
+#
+# The following parameters must be passed to this 'template':
+# - vm_name: the name of this VM for virtualbox
+# - disk_image_capacity: Size of the virtual disk image in GB
+# - vm_description: a description of this VM
+# - cimos_id: numeric ID of the operating system in the virtual box (see
+#             :data:`_CIMOS_ID`)
+_OVF_TEMPLATE = """<?xml version="1.0"?>
+<Envelope ovf:version="1.0" xml:lang="en-US" xmlns="http://schemas.dmtf.org/ovf/envelope/1" xmlns:ovf="http://schemas.dmtf.org/ovf/envelope/1" xmlns:rasd="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_ResourceAllocationSettingData" xmlns:vssd="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_VirtualSystemSettingData" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:vbox="http://www.virtualbox.org/ovf/machine">
+  <References>
+    <File ovf:id="file1" ovf:href="box.vmdk"/>
+  </References>
+  <DiskSection>
+    <Info>List of the virtual disks used in the package</Info>
+    <Disk ovf:capacity="{disk_image_capacity}" ovf:diskId="vmdisk1" ovf:fileRef="file1" ovf:format="http://www.vmware.com/interfaces/specifications/vmdk.html#streamOptimized"/>
+  </DiskSection>
+  <NetworkSection>
+    <Info>Logical networks used in the package</Info>
+    <Network ovf:name="NAT">
+      <Description>Logical network used by this appliance.</Description>
+    </Network>
+  </NetworkSection>
+  <VirtualSystem ovf:id="{vm_name}">
+    <Info>A virtual machine</Info>
+    <OperatingSystemSection ovf:id="{cimos_id}">
+      <Info>The kind of installed guest operating system</Info>
+      <Description>{vm_description}</Description>
+      <!--FIXME: this is probably not required<vbox:OSType ovf:required="false">OpenSUSE_64</vbox:OSType>-->
+    </OperatingSystemSection>
+    <VirtualHardwareSection>
+      <Info>Virtual hardware requirements for a virtual machine</Info>
+      <System>
+        <vssd:ElementName>Virtual Hardware Family</vssd:ElementName>
+        <vssd:InstanceID>0</vssd:InstanceID>
+        <vssd:VirtualSystemIdentifier>{vm_name}</vssd:VirtualSystemIdentifier>
+        <vssd:VirtualSystemType>virtualbox-2.2</vssd:VirtualSystemType>
+      </System>
+      <Item>
+        <!-- FIXME: do we need captions? -->
+        <rasd:Caption>1 virtual CPU</rasd:Caption>
+        <rasd:Description>Number of virtual CPUs</rasd:Description>
+        <!-- FIXME: make this customizable? -->
+        <rasd:ElementName>1 virtual CPU(s)</rasd:ElementName>
+        <rasd:InstanceID>1</rasd:InstanceID>
```

Comment:
KIWI machines don't boot properly with less than 2 vCPUs. I'm not sure why...

Reply:
Are you sure about that? I have booted a Leap 15 box produced from this template successfully.